.github/workflows/markdown.yml

```diff
@@ -0,0 +1,34 @@
+name: Markdown
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+  # Everyday at 9:00 AM.
+  - cron: "0 9 * * *"
```

Comment:
curious why would this need to run as a cron, if it's going to verify on each PR anyway?